pkg/reconciler/domainmapping/table_test.go

```diff
@@ -101,9 +101,9 @@ func TestReconcile(t *testing.T) {
 	}))
 }
 
-type DomainMappingOption func(dm *v1alpha1.DomainMapping)
+type domainMappingOption func(dm *v1alpha1.DomainMapping)
```

Comment:
The linter is unhappy with this change :(

Reply:
sigh. Guess I should lowercase the other functions to match

Reply:
..that should do it

Reply:
😁